client-guide/jmap-client-guide.mdown

```diff
@@ -238,20 +235,24 @@ This might return the following:
         "list": [{
             "id": "fm1u314",
             "threadId": "4f512aafed75e7fb",
-            "mailboxIds": [ "mailbox1" ],
-            "isUnread": true,
-            "isFlagged": false,
-            "isAnswered": false,
-            "isDraft": false,
+            "mailboxIds": {
+                "mailbox1": true
+            },
+            "keywords": {
+                "$seen": false,
```

Comment:
There should be nothing mapping to `false`. It either maps to `true` or it's omitted. I think let's just have `$seen` mapping to true.